data/l10n/en/main.ftl

```diff
@@ -3,7 +3,7 @@ player-description = the heroic player
 bare-hands-name = bare hands
 bare-hands-description = the harmful hands of the player
 look-at-message = You look at { INTER($object) }.
-entity-not-seen = You haven't seen { $entity }.
+entity-not-seen = There is no { $entity }.
```

Comment:
Mmh should we change look-at-message to "Try to look at …" then, because it might be there or not? Can you try to look at something that isn't there? You can certainly look *for* something that isn't there.

Reply:
Oh, the whole message is superfluous.